microdnf/context.cpp

```diff
@@ -282,10 +282,10 @@ void Context::load_rpm_repos(libdnf::rpm::RepoQuery & repos, libdnf::rpm::SolvSa
 //         try {
 //             load_rpm_repo(*repo.get());
 //             auto & solv_sack = base.get_rpm_solv_sack();
-//             // std::cout << "Loading repository \"" << repo->get_config()->name().get_value() << "\" into sack." << std::endl;
+//             // std::cout << "Loading repository \"" << repo->get_config().name().get_value() << "\" into sack." << std::endl;
```

Comment:
Please could you add todo why the code was left here?

Reply:
What TODO? I didn't comment out the code, I just updated it even though it's commented out (in an attempt to keep it working).

If you look at the first line of the comment, this is the single-threaded version that @jrohel left here, I guess mainly it htere were problems with the multi-threaded one, for debugging purposes... it's not ideal, as it will rot over time, maybe we should consider keeping the single-threaded version as an active alternative conde-path (using a compile-time or a run-time switch... maybe an env variable?)

Reply:
I know, but please could you add it to correct it. When we left some code commented out we should mark it with todo to not forget about. Thanks a lot.

Reply:
Allright, I guess I can add something.